packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/splaLicence/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dedicatedCloud_datacenter_backup_noSPLA_1": "Votre Private Cloud doit avoir une licence SPLA avant de pouvoir activer le backup. Afin d'activer la licence SPLA, veuillez vous rendre dans la section",
+  "dedicatedCloud_datacenter_backup_splaActivationLink": "Licence Windows",
+  "dedicatedCloud_datacenter_backup_noSPLA_2": "de votre produit."
```

Comment:
Could we remove camel-casing from translation ids, because that's what we follow everywhere :)